client/src/components/Chat.jsx

```diff
@@ -412,8 +416,20 @@ const Chat = () => {
             socket.off(NEW_EVENT_EDIT_MESSAGE, editMessageHandler);
         };
     }, []);
+    
+    const checkPartnerResponse = () => {
+        const currentTime = new Date().getTime();
+        if (lastMessageTime && currentTime - lastMessageTime > inactiveTimeThreshold) {
```

Comment:
make this condition cleaner by putting `currentTime - lastMessageTime > inactiveTimeThreshold` in a variable